File: BBIS.Application/DTOs/DonorScreening/DonorVitalSignsDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BBIS.Application.DTOs.DonorScreening
{
    public class DonorVitalSignsDto
    {
        public Guid? DonorVitalSignsId { get; set; }
        public Guid DonorTransactionId { get; set; }
        public Guid DonorRegistrationId { get; set; }
        public double BodyWeight { get; set; }
        public double Temperature { get; set; }
        public string BloodPressure { get; set; }
        public double PulseRate { get; set; }
        public double RespiratoryRate { get; set; }
        public double OxygenSaturation { get; set; }
        public Guid FacilitatedBy { get; set; }
        public DateTime DateOfExamination { get; set; }
        public string DonorName { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/DonorScreening/DonorVitalSignsDto.ts
export interface IDonorVitalSigns {
    DonorTransactionId: Guid,
    DonorRegistrationId: Guid,
    BodyWeight: number,
    Temperature: number,
    BloodPressure: string,
    PulseRate: number,
    RespiratoryRate: number,
    OxygenSaturation: number,
    DonorStatus: string,
    DeferralStatus: string,
    Remarks: string
}

export class DonorVitalSignsDto implements IDonorVitalSigns {
    DonorTransactionId: Guid = "";
    DonorRegistrationId: Guid = "";
    BodyWeight: number = 0;
    Temperature: number = 0;
    BloodPressure: string = "";
    PulseRate: number = 0;
    RespiratoryRate: number = 0;
    OxygenSaturation: number = 0;
    DonorStatus: string = "";
    DeferralStatus: string = "";
    Remarks: string = "";
}